it/BaseISpec.scala

```diff
@@ -95,14 +104,14 @@ abstract class BaseISpec
     ()
   }
 
-  private def clearRequestQueue(): Assertion = clearQueue("PACS001_request").status mustBe OK
+  private def clearRequestQueue(): Assertion = clearQueue("STUB001_request").status mustBe OK
 
   private def clearReplyQueue(): Assertion =
     clearQueue("PACE001_reply").status mustBe OK
 
   private def clearQueue(name: String): WSResponse =
     await {
-      wsClient.url(s"http://localhost:9324/$name?Action=PurgeQueue").get()
+      wsClient.url(s"http://localhost:9324/queue/$name?Action=PurgeQueue").get()
```

Comment:
Is the `queue` part of the url and required as the tests work this as well

Reply:
When running the tests on my MacBook I was seeing invalid request messages when it tried to purge the queue. U didn't see this on my Linux machine. It seems that adding the queue fixed the issue on the MacBook and didn't cause an issue on Linux. The exact issue I was seeing is described [here](https://github.com/softwaremill/elasticmq/issues/123).